gui/src/components/logs/Logs.svelte

```diff
@@ -47,7 +47,9 @@
         >
           {event.time.short}
         </td>
-        <td class="name">{event.name}</td>
+        <td class="name" title={event.name}
+          >{event.name.replace(/(:err$)|(:out$)/g, '')}</td
```

Comment:
Please tag this with a `SEE NOTE [LOG_COMPONENTS]` command, as this is a problem spread across many places in the code.